stock_barcodes_move_location/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Barcodes",
+    "summary": "It provides read barcode on stock operations.",
+    "version": "11.0.2.0.0",
+    "author": "Tecnativa, "
+              "Odoo Community Association (OCA)",
+    "website": "https://odoo-community.org/",
```

Comment:
Could you change at least this entry + module name following conventions?

Is it normal that copyright headers are quite various through all files ?